### PR TITLE
Enable Static File hosting

### PR DESCRIPTION
**Pull request by nanotubing**
Cleaning up configuration to enable django hosting static files from the frontend.

## Overview

This PR configures the plugins and configuration necessary for django to host static files for the frontend.

Connects #10 

## Testing Instructions

 * Check out this branch
 * Run `scripts/update`
 * Run `scripts/server`
 * In a web browser, load the [local development site](http://localhost:8181/)
 * If this test is successful, it will load a blank white screen with the title 'React App'. If it is not successful, it will load a "Page not found" 404 error

### CODE REVIEW

File: scripts/lint [hunk 1]
```diff
@@ -24,9 +24,9 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
         # Lint JavaScript
         # TODO turn this back on when there's javascript to check
```
Comment (caseycesari) [inline scripts/lint:26]: Stale comment

File: scripts/update [hunk 2]
```diff
@@ -38,6 +38,6 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
 
         # Collect Django static files
         # TODO enable this after static file hosting is corrected
```
Comment (caseycesari) [inline scripts/update:40]: Stale comment

File: src/django/iow/settings.py [hunk 3]
```diff
@@ -216,9 +213,8 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 # TODO enable static file hosting
 
-# STATIC_URL = '/static/'
-# STATIC_ROOT = os.path.join(BASE_DIR, "static")
-# STATICFILES_DIRS = ((os.path.join(STATIC_ROOT, "static")),)
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
```
Comment (caseycesari) [inline src/django/iow/settings.py:217]: When visiting the site, you should see a demo React app and not the blank screen (you can see what the demo app looks like by visiting the webpack dev server at http://localhost:4545/). The blank screen indicated that some JS files weren't being served properly. If you include the `STATICFILES_DIR` line that was commented out on line 221, it allows the JS files to be loaded properly and for the React app to load from http://localhost:8181/

File: src/django/iow/settings.py [hunk 4]
```diff
@@ -216,9 +213,8 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 # TODO enable static file hosting
```
Comment (caseycesari) [inline src/django/iow/settings.py:214]: Stale comment

**Review by caseycesari — APPROVED** [review]
Nice work! Just one quick change to get this working, and a little bit of clean-up.

File: src/django/iow/settings.py (hunk 3 above)
Comment (nanotubing) [reply src/django/iow/settings.py:217]: Thanks for catching this. I couldn't figure out if this was necessary or not, and after checking a few different projects it looked like some did and some did not have it, so I figured I'd try without it. Lesson learned.

**Pull request merged**